ft_lib/ft_range.py

```diff
@@ -0,0 +1,12 @@
+def ft_range(stop, start=0, step=1):
+    if isinstance(stop, int) and isinstance(start, int) and isinstance(step, int):
+        l = []
```

Comment:
Rename please variable. I don't understand what it means. For example **res** or **l_res**